terminology.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Terminology cheat sheet for Kokkos
+---
+
+<!--#ifndef PRINT-->
+
+# Terminology cheat sheet
+
+<!--#endif-->
+
+## Memory
+
+| Kokkos                                        | Cuda                         | HIP                | SYCL                        |
+|-----------------------------------------------|------------------------------|--------------------|-----------------------------|
+| `Kokkos::DefaultExecutionSpace::memory_space` | Global memory                | Global memory      | Global memory               |
+| `Kokkos::ScratchMemorySpace`                  | Shared memory                | Shared memory      | (Shared) local memory       |
+| --                                            | Local memory                 | Local memory       | --                          |
+| --                                            | Register                     | Register           | Private memory              |
```

Comment:
```suggestion
```

Reply:
I agree, as discussed, I think we can only keep lines that have Kokkos equivalent